daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/RuntimeData.scala

```diff
@@ -689,6 +698,10 @@ sealed class ElementRuntimeData(
       name
     }
   }
+
+  override def namespace: NS = dpathElementCompileInfo.namedQName.namespace
+  override def optNamespacePrefix: Option[String] = dpathElementCompileInfo.namedQName.prefix
+  override def sscd: String = dpathElementCompileInfo.sscd
```

Comment:
I've forgotten what does sscd stands for. Which makes me thinks it's used rarely enough that maybe it needs a more descriptive name? Could be done in another PR, since it's not related to this.